src/modules/class.js

```diff
@@ -0,0 +1,24 @@
+export const list = JSON.parse(localStorage.getItem('list')) || [];
+
+export class ToDoTasks {
+  constructor(description = '', completed = false, index = null) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+export const toDoTasks = new ToDoTasks();
+
+export const updateLocalStorage = () => {
+  const newLists = document.querySelectorAll('li');
+  const list = [];
```

Comment:
- [ ] I discovered that the page must be refreshed after adding in the first Item in order to add the next items to the list. Please make sure to fix this issue.
- [ ] Items added to the list get disappeared after refreshing the page. Please let's make sure that all todos are saved into the local storage correctly. From the requirements of this project, all changes to the To-Do List should be saved in local storage.

Reply:
Fixed